Makefile

```diff
@@ -38,19 +50,42 @@ security: db
 
 .PHONY: spec
 spec: db
+ifeq ($(ENV_ARG), dev)
+	@$(BASH_DEV) "bin/rails spec"
+else
 	@$(BASH_TEST) "bin/rails spec"
+endif
 
 .PHONY: up
 up: db
 	@$(COMPOSE_DEV) up
 
 .PHONY: rebuild
-rebuild:
-	@$(COMPOSE_DEV) down
-	@$(COMPOSE_DEV) build
+rebuild: down build
+
+.PHONY: build
+build:  ## Builds the service
+ifeq ($(ENV_ARG), dev)
+	$(COMPOSE_DEV) build
+else
+	$(COMPOSE_TEST) build
+endif
+
+.PHONY: down
+down:  ## Stops all docker services
+ifeq ($(ENV_ARG), dev)
+	@$(COMPOSE_DEV) $(DOWN)
+else
+	@$(COMPOSE_TEST) $(DOWN)
+endif
 
 .PHONY: clean
 clean:
 	rm -r data || true
+ifeq ($(ENV_ARG), dev)
+	$(COMPOSE_DEV) run gibct rm -r coverage log/* tmp || true
```

Comment:
Do you want the clean command to remove volumes as well? I see it's just removing coverage, log/* and tmp

Reply:
I'm not sure, this was an existing command that was updated to include the env option

Reply:
Additionally, you may want a true clean command `@$(COMPOSE_DEV) down --rmi all --volumes` which will remove any images or volumes hanging around

Reply:
I tested out adding that as https://github.com/department-of-veterans-affairs/vets-api/blob/master/Makefile#L86 but because it blew away all images including some of those used by vets-api I decided to not include the command as vets-api is the more used repo and wanted to reduce people affecting their local vets-api when working in gibct-data-service